src/negotiation.rs

```diff
@@ -473,7 +494,7 @@ where
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         let encoded = base58_monero::encode_check(consensus::serialize(self).as_ref())
             .expect("Encoding in base58 check works");
-        write!(f, "{}", encoded)
+        write!(f, "Offer{}", encoded)
```

Comment:
I would add a delimiter, like a `:`, between different character encodings. Cashaddresses for example do this and even use it for visible versioning.

Reply:
I hesitated to add the `:`, now its done :+1: